### PR TITLE
Merge Dv/master to master

### PR DESCRIPTION
**Pull request by TechedLemur**
Any reason why we still have a DV/master branch and not merge this, especially when Analyze only uses the "master"-branch (if you don't manually edit the json-file), and only DV uses the unpacker.

### CODE REVIEW

**Comment by TheLarsinator** [comment]
EV also uses the unpacker for our inverter messages!

**Comment by TechedLemur** [comment]
> EV also uses the unpacker for our inverter messages!

Ok, forgot that we are going for I19

But the messages should not collide, so why not have one master?

File: unpacker.json [hunk 1]
```diff
@@ -4357,48 +4357,6 @@
...
-		"ECU_Torque_Allocation": {
-			"type": "Uint",
-			"startbyte": 2,
-			"length": 1,
-			"factor": 1.0,
-			"displayname": "",
-			"unit": "",
-			"expectedmin": 0.0,
-			"expectedmax": 0.0
-		},
```
Comment (JosteinBrovold) [inline unpacker.json:4380]: Wouldnt removing these destroy old Atmos / ELD logs? As Im aware, you also need the correct unpacker for the logs stored from previous years?

**Comment by TechedLemur** [comment]
Closing until new logstore is released

File: unpacker.json (hunk 1 above)
Comment (TechedLemur) [reply unpacker.json:4380]: As of now, yes. So we will wait a bit with this I think.

**Pull request closed without merging**